modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -47,9 +47,13 @@ topk(10,count by (job)({__name__=~".+"}))
 
 ** *If the metrics relate to a core {product-title} project*, create a Red Hat support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
 
-. Check the TSDB status in the Prometheus UI.
-.. In the *Administrator* perspective, navigate to *Networking* -> *Routes*.
-.. Select the `openshift-monitoring` project in the *Project* list.
-.. Select the URL in the `prometheus-k8s` row to open the login page for the Prometheus UI.
-.. Choose *Log in with OpenShift* to log in using your {product-title} credentials.
-.. In the Prometheus UI, navigate to *Status* -> *TSDB Status*.
+. Review the TSDB status using the Prometheus HTTP API by running the following commands as a cluster administrator:
++
+[source,terminal]
+----
+$ oc login -u <username> -p <password>
+$ host=$(oc -n openshift-monitoring get route prometheus-k8s -ojsonpath={.spec.host})
+$ token=`oc sa get-token prometheus-k8s -n openshift-monitoring`
```

Comment:
if we use this way to get token,then `$ oc login -u <username> -p <password>` should be removed

Reply:
`oc sa get-token prometheus-k8s -n openshift-monitoring` the command need admin access, should keep `login `command and get token this way `token=$(oc whoami -t)`

Reply:
Changed the get token method to `token=$(oc whoami -t)`